pre_commit/languages/docker.py

```diff
@@ -76,7 +78,29 @@ def install_environment(
         os.mkdir(directory)
 
 
+def _docker_is_rootless() -> bool:
+    returncode, stdout, stderr = cmd_output(
+        'docker', 'system', 'info',
+    )
+    for line in stdout.splitlines():
+        # rootless docker has "rootless"
+        # rootless podman has "rootless: true"
+        if line.strip().startswith('rootless'):
+            if 'false' not in line:
+                print(line)
```

Comment:
`print()` shouldn't be used for logging (unless you're debugging something right now)

Reply:
this was debugging.